src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBindingContext.cs

```diff
@@ -52,5 +60,10 @@ public TraceWriter Trace
         {
             get { return _trace; }
         }
+
+        /// <summary>
+        /// The short name of the current function. 
+        /// </summary>
+        public string MethodName { get; private set; }
```

Comment:
Shucks.  Looks like it went public with this change: 605f161fa1594bd61b076271287ebccb2d373f29 . I wish it was still internal.  
It's still got the wrong  shape.  It's got a FullName, ShortName, neither are which we want ... we need Method.Name.  I think exposing this class here would get confusing. 

---
In reply to: [117294645](https://github.com/Azure/azure-webjobs-sdk/pull/1148#discussion_r117294645) [](ancestors = 117294645)